[2024-02-02 04:13:26,239] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:13:21.812451+00:00 [queued]>
[2024-02-02 04:13:26,244] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:13:21.812451+00:00 [queued]>
[2024-02-02 04:13:26,245] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 04:13:26,245] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-02 04:13:26,245] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 04:13:26,251] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): import_file_to_mysql> on 2024-02-02 04:13:21.812451+00:00
[2024-02-02 04:13:26,254] {standard_task_runner.py:52} INFO - Started process 869 to run task
[2024-02-02 04:13:26,257] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'import_file_to_mysql', 'manual__2024-02-02T04:13:21.812451+00:00', '--job-id', '311', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp3q5sjbky', '--error-file', '/tmp/tmpl3e4i_x7']
[2024-02-02 04:13:26,257] {standard_task_runner.py:80} INFO - Job 311: Subtask import_file_to_mysql
[2024-02-02 04:13:26,287] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:13:21.812451+00:00 [running]> on host c3bb523480bf
[2024-02-02 04:13:26,327] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=import_file_to_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-02-02T04:13:21.812451+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-02T04:13:21.812451+00:00
[2024-02-02 04:13:28,708] {logging_mixin.py:115} INFO - Successfully install pandas
[2024-02-02 04:13:31,317] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-02 04:13:33,608] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-02-02 04:13:33,704] {python.py:173} INFO - Done. Returned value was: import to mysql
[2024-02-02 04:13:33,719] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=import_file_to_mysql, execution_date=20240202T041321, start_date=20240202T041326, end_date=20240202T041333
[2024-02-02 04:13:33,770] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 04:13:33,791] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
